src/components/entity-panel/sections/entity/wows/TheEntityListContent.vue

```diff
@@ -0,0 +1,193 @@
+            @click="entityChipGroupOnChangeHandler"
+          />
+         </v-item>
+      </v-item-group>
+    </v-lazy>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import Component from 'vue-class-component'
+import { Api, Entity, Game } from '@/store/types'
+import { AppRoomGetters } from '@/store/modules/app/room'
+import { namespace } from 'vuex-class'
+import { Namespaces } from '@/store'
+import { Ship, EntitiesDataApi } from '@/types/Games/Wows'
+import { AppToolGetters } from '@/store/modules/app/tools'
+import { Tool } from '@/tools/Tool'
+import EntityCard from '../EntityCard.vue'
+import { GameApiRoutes } from '@/games/types'
+import { Prop, Watch } from 'vue-property-decorator'
+import EntitySearch from '../EntitySearch.vue'
+import { CanvasElement, CanvasElementType } from '@/types/Canvas'
+import { SocketCanvasGetters } from '@/store/modules/socket/canvas'
+
+const AppRoom = namespace(Namespaces.APP_ROOM)
+const AppTools = namespace(Namespaces.APP_TOOLS)
+const SocketCanvas = namespace(Namespaces.SOCKET_CANVAS)
+
+@Component({
+  name: 'TheEntityListContent',
+  components: {
+    EntityCard,
+    EntitySearch
+  }
+})
+export default class TheEntityListContent extends Vue {
+  @Prop() game!: Game
+  @AppRoom.Getter(AppRoomGetters.API) api!: Api[];
+  @AppTools.Getter(AppToolGetters.ENABLED_TOOL) enabledTool!: Tool | undefined
+  @SocketCanvas.Getter(SocketCanvasGetters.CANVAS_ELEMENTS) canvasElements!: CanvasElement[]
+
+  @Watch('enabledTool')
+  onPropertyChangeHandler () {
+    if (this.enabledTool?.name !== 'entity') {
+      this.selectedDefaultEntityId = -1
+      this.selectedEntityId = -1
+    }
+  }
+
+  search = ''
+  selectedDefaultEntityId = -1
+  selectedEntityId = -1
+
+  get entities (): Entity[] {
+    const apiData: Api | undefined = this.api.find((api: Api) => api.name === GameApiRoutes[Game.WOWS].entities)
+    if (apiData && apiData.data) {
+      const entities = (apiData.data as EntitiesDataApi).entities as Ship[]
+      return entities.filter((ship: Ship) => !ship.default).map((ship: Ship): Ship => ({
+        id: ship.id,
+        uuid: ship.uuid,
+        name: ship.name,
+        title: ship.title,
+        game: Game.WOWS,
+        image: ship.image,
+        tier: ship.tier,
+        type: ship.type,
+        default: ship.default,
+        data: ship.data,
+        canvasImage: ship.canvasImage,
+        color: ship.color
+      }))
+    }
+    return []
+  }
+
+  get entitySearch (): Entity[] {
+    if (!this.search || this.search?.length < 3) {
+      return []
+    } else {
+      return this.entities.filter((entity: Entity) => entity.name.toLowerCase().includes(this.search.toLowerCase())).sort((a: Entity, b: Entity) => a.name > b.name ? 1 : -1)
+    }
+  }
+
+  get getDefaultEntities (): Entity[] {
+    const apiData: Api | undefined = this.api.find((api: Api) => api.name === GameApiRoutes[Game.WOWS].entities)
+    if (apiData && apiData.data) {
```

Comment:
if (apiData?.data)

Reply:
I actually want know why this was resolved without changes, as I would point totally same thing. ☝️